### ISSUE TITLE
Make it original

### ISSUE THREAD

**Issue by davelab6** [erikdkennedy/figtree#1]
I encourage you to differentiate this from Product Sans and Google Sans, so that it could become available in Google Fonts :)

**Comment by erikdkennedy** [comment]
Nice – will make this a design goal going forward. I'll check in when I've done that more, but probably before adding italics.

**Comment by erikdkennedy** [comment]
Hi Dave,
 
Here are some of the changes I've made to Blueberry to distinguish it from Google Sans. Some are more conceptual, some are more specific.

<img width="700" alt="blueberry-google-comparison" src="https://user-images.githubusercontent.com/3440839/158866027-d9354761-b054-4f25-953d-b6c12b62f5ad.png">

- Since /a and /g are perhaps the best glyphs to differentiate geometrics, I made both more distinct. /a now has a flat-topped bowl and /g has a truncated descender with a terminal consistent with /jy
- Product Sans has a motif of angled terminals just overshooting the cap height (or baseline), for glyphs like 6, 7, and 9. Blueberry uses horizontal terminals with no overshoot for all of those.
- /y has a curved descender consistent with /fgjt 
- /M has a sharp midpoint and /W's midpoint sits on the cap height
- /ARP have lowered crossbars (more visible in bold), which makes the caps feel more solid and serious
- The punctuation varies a bit, including the non-furled question mark, stroke form for the comma, and unbroken ampersand

Is this enough differentiation, or should I push farther from Product/Google Sans? If the latter, how close is this to being acceptably different for Google Fonts?

(Sorry for being obnoxious here – this is my first font, and I'm not sure what's deemed acceptable in the somewhat crowded world of geometric sans) 
 
Thanks,
Erik
